nexus-sdk/src/main/java/io/nexusrpc/OperationNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package io.nexusrpc;
+
+/** An operation was not found for an ID. */
+public class OperationNotFoundException extends Exception {
```

Comment:
There's a bunch of other error types in the spec. I would use the same approach I took in Go: https://pkg.go.dev/github.com/nexus-rpc/sdk-go@v0.0.9/nexus#HandlerError

Reply:
These are exceptions meant to be thrown by the user. They can/will be translated to/from the internal/serializable representations internally. We can consider extracting a common base class if needed when we get to the client side, but people need to be able to throw these classes explicitly, not some generic class w/ an enum. This is another one of those cases where matching Java norms is more important than the Go approach.

Reply:
I'd also support a single base class for handler errors and subclasses per type but the full set of handler error types should be in the SDK in some form.

Reply:
I am not sure yet what the caller side looks like, which is why I said we can extract when we get there. I don't want to guess about that until we get there.